backend/organization/views/project_views.py

```diff
@@ -1066,6 +1064,33 @@ def get_queryset(self):
         return open_membership_requests
 
 
+class HasUserRequested(APIView):
+    def get(self, request, url_slug):
+        if not request.user.is_authenticated:
```

Comment:
If you define a permission class before the get request this would automatically be covered. https://www.django-rest-framework.org/api-guide/permissions/#isauthenticated